app/models/invitation.rb

```diff
@@ -37,4 +43,9 @@ def user_already_in_system
       errors.add(:email, "User already registered in system")
     end
   end
+
+  def week_notification
+    date = 1.week.from_now.beginning_of_day
```

Comment:
@shir I agree, but this decision - part of the solution about unsubscriptions. Please see my next PR https://github.com/metastudio/cashbox/pull/409 and you will understand what I mean.